GREEDY/ImagePreparation/ImageFormating.cs

```diff
@@ -42,10 +42,16 @@ private Bitmap Crop(Bitmap bitmap, double[] cornerPoints)
         //returns the bigest found rectangle
         private Bitmap FilterCropedImages(List<Bitmap> list)
         {
-            if (list.Any())
-                return list.MaxBy(x => x.Height * x.Width); // this guy throws an exception if list is null.
-            else
-                return null;
+            try
+            {
+                if (list.Any())
+                    return list.MaxBy(x => x.Height * x.Width); // this guy throws an exception if list is null.
```

Comment:
Just like to point out that we are using the microsoft coding standard which i believe requires { } clauses and new lines even when there is one line in the block (easier to read that way)

Reply:
Can you show me what is wrong there? 😅 sorry, but I cannot understand. I just added try and catch. No other changes was made